vignettes/teal-options.Rmd

```diff
@@ -84,10 +84,16 @@ This indicates whether to print the `JavaScript` console logs to the `R` console
 
 Default: `FALSE`.
 
-### `teal.renv.lockfile` (`character`)
+### `teal.lockfile.enable` (`logical`)
 
-The path to the pre-computed `renv` lockfile that will be shared through teal app. To read more about lockfile usage creation check `?teal::teal_lockfile`.
+This enables app users to download `renv` lockfile by clicking "download lockfile" in the footer. If 
```

Comment:
```suggestion
This enables app users to download `renv` lockfile by clicking `"download lockfile"` in the footer of the app. If 
```

Reply:
I would discourage usage of `"app"` in the documentation - use `"application"` instead

Reply:
updated the comment